build.sh

```diff
@@ -411,8 +411,6 @@ function install_halide() {
       WITH_METAL= \
       WITH_EXCEPTIONS=1 \
       make -f ../Makefile -j $CORES install || exit 1
-      mkdir -p ${INSTALL_PREFIX}/include/Halide
-      mv ${INSTALL_PREFIX}/include/Halide*.h  ${INSTALL_PREFIX}/include/Halide/
```

Comment:
You missed the `-d "${INSTALL_PREFIX}/include/Halide"` a couple of lines up.
This should check for the `Halide.h` _file_ now.

Reply:
thanks for catchnig it, fixed